flyteidl/protos/flyteidl/admin/agent.proto

```diff
@@ -160,8 +160,18 @@ message GetTaskLogsRequest {
   bytes resource_meta = 2;
 }
 
+message GetTaskLogsResponseHeader {
+  string token = 1;
+}
+
+message GetTaskLogsResponseBody {
+  repeated string results = 1;
+}
+
 // A response containing the logs for a task execution.
 message GetTaskLogsResponse {
-  // The execution log results.
-  string results = 1;
+  oneof part {
+    GetTaskLogsResponseHeader header = 1;
```

Comment:
Does the response pattern end up looking like 
```
GetTaskLogsResponse{Header{token=3}}
GetTaskLogsResponse{Body{results=[...]}} // 3 log lines
GetTaskLogsResponse{Header{token=9}
GetTaskLogsResponse{Body{results=[...]}} // 6 log lines
```

And then a future request call would include `token=9`?